mobile-version.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css?family=Poppins:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i&amp;display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style-mobile.css">
+    <title>Mobile Version</title>
+</head>
+<body>
+    <main>
+        <section class = "works">
+            <div class="job-1">
+                <img src="/images/Snapshoot-Portfolio.jpg" alt="job-1 image">
+                <div class="content">
+                    <h3>Tonic</h3>
+                    <ul>
+                        <li>Canopy</li>
+                        <li>Back End DEv</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>A daily selection of privately <br> personalized reads; no accounts or sign-ups required.</p>
+                    <ul>
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">javascript</a></li>
+                    </ul>
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+            <div class="job-2">
+                <img src="/images/Snapshoot-Portfolio-2.jpg" alt="job-2 image">
+                <div class="content">
+                    <h3>Multi-Post Stories</h3>
+                    <ul>
+                        <li>Canopy</li>
+                        <li>Back End DEv</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul>
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">javascript</a></li>
+                    </ul>
+                    <button type="button">See Project</button>
```

Comment:
- [ ] I can see here that you didn't made the Hover and disabled and pressed effects CSS on the Button please add them to match the design👌🏻💯


- [ ] Also Please put the same padding and margin for the button👌🏻